packages/web/app/src/lib/urql.ts

```diff
@@ -40,7 +40,10 @@ export const urqlClient = createClient({
         GraphQLInputField: noKey,
         GraphQLArgument: noKey,
         SchemaCoordinateUsage: noKey,
+        SuccessfulSchemaCheck: ({ id }) => `SchemaCheck:${id}`,
+        FailedSchemaCheck: ({ id }) => `SchemaCheck:${id}`,
```

Comment:
```suggestion
```

B−

Reply:
I would still prefer to have the type name in the cache key name, so it is easier to spot it ;)

Reply:
:D